tests/e2e/targetallocator-features/00-install.yaml

```diff
@@ -51,6 +51,7 @@ spec:
     securityContext:
       runAsUser: 1000
       runAsGroup: 3000
+      fsGroup: 3000
```

Comment:
i believe you need to set this in a different block for the podSecurityContext

Reply:
Is it good now?